src/components/Tokens/TokenRowWallets.tsx

```diff
@@ -39,7 +39,7 @@ type TokenRowWalletsProps = {
   isExecutable: boolean | undefined;
 };
 
-type actionType = "PRICE" | "BALANCE";
+export type actionType = "PRICE" | "BALANCE";
```

Comment:
as a good pracice can we find some spot in repo to store this kind of shared types/interfaces?
I believe there should already be one for that, take a look around the repo

Reply:
done moved to interfaces folder inside portfolio.